b/bottleneck/bottleneck_1.3.5_ubi_9.3.sh

```diff
@@ -22,29 +22,27 @@
 PACKAGE_NAME=bottleneck
 PACKAGE_VERSION=${1:-v1.3.5}
 PACKAGE_URL=https://github.com/pydata/bottleneck.git
-
+    
 # Install dependencies and tools.
 yum install -y wget gcc gcc-c++ gcc-gfortran git make  python-devel  openssl-devel 
-
+    
 #clone repository 
 git clone $PACKAGE_URL
 cd  $PACKAGE_NAME
 git checkout $PACKAGE_VERSION
-
-pip install .
-
+    
 #install
-if ! (python3 setup.py install) ; then
+if ! (pip install .) ; then
```

Comment:
Why we need to just do pip install ... this will just install dependencies from requirement.txt